### PR TITLE
Remove non existing fields from classifiers - IAM.

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Remove non-existing fields from classifiers - IAM.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by dantavori** [comment]
classifiers that don't contain "User Profile" in their names aren't IAM classifiers.
and most (if not all) of the fields exist in [IAM pack](https://github.com/demisto/content-premium/tree/master/Packs/IAM/IncidentFields) or in [CommonTypes](https://github.com/demisto/content/tree/master/Packs/CommonTypes) pack. can you go through them again?

File: Packs/Active_Directory_Query/Classifiers/classifier-User_Profile_-_Active_Directory_(Incoming).json [hunk 1]
```diff
@@ -6,10 +6,6 @@
   "User Profile": {
    "dontMapEventToLabels": true,
    "internalMapping": {
-    " Street Address": {
```
Comment (dantavori) [inline Packs/Active_Directory_Query/Classifiers/classifier-User_Profile_-_Active_Directory_(Incoming).json:9]: the space before the "Street" here should be removed but streetaddress incident field exist (under CommonTypes)

Comment (ChanochShayner) [reply Packs/Active_Directory_Query/Classifiers/classifier-User_Profile_-_Active_Directory_(Incoming).json:9]: there is already a Street address field in this classifier

**Comment by ChanochShayner** [comment]
@dantavori I went through the fields again, (I returned some of the fields after your explanation).
The fields I just removed do not exist in common types or in IAM.
I would appreciate it if you could go over it and confirm.
Thanks!

File: Packs/Okta/ReleaseNotes/2_1_4.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Classifiers
+##### User Profile - Okta (Outgoing)
+- Maintenance and stability enhancements.
```
Comment (guykeller) [inline Packs/Okta/ReleaseNotes/2_1_4.md:4]: Here and everywhere else, let's avoid using this line and instead write down what actually happened

**Review by guykeller — APPROVED** [review]
One small comment, looks good other than that

File: Packs/Okta/ReleaseNotes/2_1_4.md (hunk 2 above)
Comment (ChanochShayner) [reply Packs/Okta/ReleaseNotes/2_1_4.md]: done

**Review by ohaim1008 — APPROVED** [review]

**Pull request merged**